diagnostic.md

```diff
@@ -62,7 +62,9 @@ var result = strangeAdd(0, -5);
 
 What is the value of result?  Explain your answer.
 
-_Replace this text with your answer._
+no result is passed, since both expressions value to false. a is with 0 not
+bigger than 0 and b is with -5 not bigger than 0, so the if statement won't run,
+so no return will be passed.
```

Comment:
Since neither of the arguments are true, it will skip the `if` loop. This moves it to the `return a - b` value. Thus 0 - (-5) = 5